src/messages/formatters.ts

```diff
@@ -222,3 +222,20 @@ export const TimestampStyles = {
  * The possible values, see {@link TimestampStyles} for more information.
  */
 export type TimestampStylesString = typeof TimestampStyles[keyof typeof TimestampStyles];
+
+export const Faces = {
```

Comment:
Any reason why this isn't an enum?

Reply:
For the same reason that TimestampStyles isn't I guess, I just went with what was already being used for the sake of consistency.

Reply:
`TimestampStyles` isn't an enum because TypeScript's developers didn't want to bother making values without the enum work for over 6 years. (ಠ_ಠ)>⌐■-■

That's not the case here, you can use enum for those faces. (⌐■-■)

Reply:
Alright should be resolved now!